tools/install/operator.py

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env python3
+
+import os
+import glob
+
+## == Constant Definitions ==##
+TAB = " " * 4
+
+top_dir = os.path.abspath(os.path.join(os.path.dirname(__file__), "../.."))
+
+def guess_solib_dir():
+    dirs = glob.glob(os.path.join(top_dir, "bazel-bin/_solib_*"))
+    assert len(dirs) == 1, "Only one _solib_* directory under bazel-bin is expected"
+    return dirs[0]
+
+solib_dir = guess_solib_dir()
+
+def install_local_solibs():
+    relative_dir = solib_dir[len(top_dir)+1:]
+    for lib in os.listdir(solib_dir):
+        if os.path.isfile(os.path.join(solib_dir, lib)):
+            print("{}install(\"{}/{}\", \"lib/{}\")".format(TAB, relative_dir, lib, lib))
+
+def install_data(src, dst):
+    print("{}install(\"{}\", \"{}\")".format(TAB, src, dst))
+ 
+def install_cyber():
+    cyber_conf = "cyber/conf/cyber.pb.conf"
+    install_data(cyber_conf, cyber_conf)
+    dreamview_conf = "cyber/conf/dreamview_sched.conf"
+    install_data(dreamview_conf, dreamview_conf)
+    install_data("bazel-bin/cyber/mainboard/mainboard", "bin/mainboard")
+
+def install_cyber_examples():
+    # Common Component Example
+    install_data("bazel-bin/cyber/examples/common_component_example/libcommon_component_example.so",
+            "cyber/examples/common_component_example/libcommon_component_example.so")
+    install_data("cyber/examples/common_component_example/common.dag", "cyber/examples/common_component_example/common.dag")
```

Comment:
Hardcoding these libraries and configs and data is really tiresome, and this is why I say that we should find another way to handle this.  Bazel extensions written in Starlark was the perfect choice to do such tasks.

Will revisit later. Ref: https://github.com/RobotLocomotion/drake/blob/master/tools/install/install.bzl

Reply:
Done.